experimental/Schemes/IdealSheaves.jl

```diff
@@ -730,6 +740,7 @@ function match_on_intersections(
     end
 
 ## make sure we are working on consistent data
+    @check
     if check
```

Comment:
Huh, shouldn't this be
```suggestion
    @check begin
```
(Genuine question, I forgot how we did it in the end, guess I should look at the `@check` docu...)

Reply:
Yes. Corrected.